### DISCUSSION TITLE
notes/java/jvm

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#652] [Announcements]
# notes/java/jvm

回顾一下字节码 在 Java 中，JVM 可以理解的代码就叫做字节码（即扩展名为 .class 的文件），它不面向任何特定的处理器，只面向虚拟机。Java 语言通过字节码的方式，在一定程度上解决了传统解释型语言执行效率低的问题，同时又保留了解释型语言可移植的特点。所以 Java 程序运行时比较高效，而且，由于字节码并不针对一种特定的机器，因此，Java...

http://localhost:8080/notes/java/jvm.html

**Comment by cp-gege** [comment]
你好
